lib/Mojolicious/Validator/Validation.pm

```diff
@@ -148,6 +157,13 @@ Name of field currently being validated.
 
 L<Mojolicious::Validator> object this validation belongs to.
 
+=head2 is_optional
+
+  my $is_optional = $validation->is_optional;
+  $validation     = $validation->is_optional('foo');
+
+Flag, indicating optionality of field currently being validated.
```

Comment:
It's very confusing to read the code where you store ```$name```in is_optional(), the example code takes a string, while the documentation says it's a flag.